portugues/revisa/tenta-consegue/sem-resposta/contato-desenvolvedor.md

```diff
@@ -0,0 +1 @@
+Voc� pergunta para o desenvolvedor quando a nova versao do aplicativo estar� dispon�vel para download, por�m ele responde que nao est� mais envolvido com o mesmo.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.